webapi/src/main/scala/org/knora/webapi/store/triplestore/upgrade/RepositoryUpdater.scala

```diff
@@ -72,6 +74,25 @@ class RepositoryUpdater(
     log = log
   )
 
+  private val tempDirNamePrefix: String = "knora"
+
+  /**
+    * Deletes previously created temp directories if these were not deleted after update.
+    * It could happen if update crashed before ends.
+    */  
```

Comment:
In my opinion there is too much logic in this docstring. I would just describe the method with what it is actually doing, something like: "Deletes directories inside temp directory starting with tempDirNamePrefix."

Reply:
True, created docstring before moving filename prefix to the `tempDirNamePrefix`.